.github/workflows/test-unit.yaml

```
@@ -0,0 +1,32 @@
+name: test-unit
```

Shall we call as follows to make it consistent with Trainer: https://github.com/kubeflow/trainer/blob/master/.github/workflows/test-python.yaml#L1

So it would be easier for developers to find similar tests 

```suggestion
name: Unit Test - Python
```